python-sdk/src/astro/sql/operators/append.py

```diff
@@ -68,7 +68,7 @@ def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
             target_table=self.target_table,
             source_to_target_columns_map=self.columns,
         )
-        context["ti"].xcom_push(key="append_query", value=db.sql)
+        context["ti"].xcom_push(key="append_query", value=str(db.sql))
```

Comment:
Have you tested this?

Can we add a test to check the type / contents of what is pushed

Reply:
Yes we tested it on the DAG

Reply:
yes, 
<img width="1475" alt="Screenshot 2022-10-14 at 8 03 07 PM" src="https://user-images.githubusercontent.com/98807258/195872679-9ec15adb-7314-4b95-961f-4effb98dcc59.png">

Reply:
We also checked the type and it is `str`

Reply:
sounds good

Reply:
We should also a add a unit-test in a follow-up PR next week